app/foodzone/venues/address.rb

```diff
@@ -0,0 +1,21 @@
+module Venues
+  class Address
+    def self.from_str(address)
+      street, number, hood = address.split(',')
+        .map { |val| val.gsub(/^\s+|\s+$/,'') }
+
+      new(street, number, hood)
+    end
+
+    attr_reader :street, :number, :hood
+    def initialize(street, number, hood)
+      @street = street
+      @number = number
+      @hood = hood
+    end
+
+    def to_s
+      [street, number, hood].join(', ')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.